common/rpc/interceptor/telemetry.go

```diff
@@ -98,7 +98,7 @@ func (ti *TelemetryInterceptor) Intercept(
 	// if the method name is not defined, will default to
 	// unknown scope, which carries value 0
 	scope, _ := ti.scopes[methodName]
-	scope = ti.overrideScope(scope, methodName, req)
+	scope = ti.overrideScope(scope, req)
```

Comment:
Can you give these vars better names? Something like:
1. `scope` -> `scopeDef`
2. `metricsScope` -> `namespaceScope`.

Reply:
`ti.scopes` probably should be `ti.scopeDefs` too.